### DISCUSSION TITLE
Test addresses

### DISCUSSION THREAD

**Discussion by cenwadike** [aurora-is-near/rainbow-bridge#743] [Q&A] [answered]
currently trying to implement some custom test on [rainbow-bridge-connector](https://github.com/aurora-is-near/rainbow-token-connector/tree/master/bridge-token-factory/src), I'd like to know if the address for [token_locker](https://github.com/aurora-is-near/rainbow-token-connector/blob/master/bridge-token-factory/src/lib.rs#:~:text=fn%20token_locker()%20%2D%3E,%7D) as well as other eth(hex) test addresses are random.

**Comment by mfornet** [comment, marked as answer]
@sept-en is there a curated list with addresses of all deployed contracts?

All address are public; and the source code is available and verified whenever possible. In particular the two you are asking are:
- [factory.bridge.near](https://explorer.near.org/accounts/factory.bridge.near)
- Each token, has its own account id. It can be deduced, by the original address on ethereum concatenated with factory account: For example: https://explorer.near.org/accounts/dac17f958d2ee523a2206206994597c13d831ec7.factory.bridge.near . You can see here a subset of the contracts deployed: [TokenList](https://github.com/aurora-is-near/bridge-assets/blob/master/assets/aurora.tokenlist.json)
